api/base/serializers.py

```diff
@@ -84,7 +84,7 @@ def _url_val(val, obj, serializer, **kwargs):
     else:
         url = val
 
-    if not url:
+    if not url and not url == 0:
```

Comment:
This is fine, but `url != 0` is preferred for readability. Not a blocker for merge.